ddp-workspace/projects/ddp-angio/src/app/components/data-release/data-release.component.ts

```diff
@@ -52,6 +52,18 @@ import { WindowRef } from 'ddp-sdk';
                                 <p class="PageContent-text" translate>
                                     Toolkit.DataRelease.MainText.Section3
                                 </p>
+
+                              <div class="row NoMargin Left">
+                                <a href="http://em.rdcu.be/ls/click?upn=1VX9wGiUV7k-2FG8imEHteF25Ol5yBjR8SA-2FhQtoXbTgU-3DgnoE_eYO6zdaQoUubxrJz-2BH2Zv38FHa3PI5OJUb9BGSLB-2BF6glT83ifBpYralVEDHtm8fUe-2Fn2THVk97Rxygjq3vwNUApHjpwIHPbAn8NGj-2B4k4IRyGmAQwVgdtIxzIPwH4I5t0r5oau6ZlrcVrlqp9-2FKg4-2FwNLNPuZX3HycTX0v1XpUk2lh15XEgnM6M0A5rOIG1YGhUUshXmSFLbpMQGgGKsQb6ejoYL1PsmMQEhEJu94850We9c0axfNJM1lsMxWGnC83WszbjBFZwZBjJ5KrqH09R57ssi7kp2rpSMCUNxrEZfn-2BdVj0614MEOChUvTOT"
+                                   style="margin-top:20px;" >
```

Comment:
```suggestion
                                <a href="http://em.rdcu.be/ls/click?upn=1VX9wGiUV7k-2FG8imEHteF25Ol5yBjR8SA-2FhQtoXbTgU-3DgnoE_eYO6zdaQoUubxrJz-2BH2Zv38FHa3PI5OJUb9BGSLB-2BF6glT83ifBpYralVEDHtm8fUe-2Fn2THVk97Rxygjq3vwNUApHjpwIHPbAn8NGj-2B4k4IRyGmAQwVgdtIxzIPwH4I5t0r5oau6ZlrcVrlqp9-2FKg4-2FwNLNPuZX3HycTX0v1XpUk2lh15XEgnM6M0A5rOIG1YGhUUshXmSFLbpMQGgGKsQb6ejoYL1PsmMQEhEJu94850We9c0axfNJM1lsMxWGnC83WszbjBFZwZBjJ5KrqH09R57ssi7kp2rpSMCUNxrEZfn-2BdVj0614MEOChUvTOT"
                                   target="_blank"
                                   style="margin-top:20px;">
```
All external links should be opened in the new tab.